config/machine-api-operator-deployment.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: machine-api-operator
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: machine-api-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: machine-api-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: machine-api-operator
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: machine-api-operator
+      containers:
+      - name: kube-rbac-proxy
+        image: quay.io/openshift/origin-kube-rbac-proxy:4.2.0
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
```

Comment:
this will use an at startup generated self-signed cert, is that intended?

Reply:
yes, and this is re;ated, https://github.com/openshift/machine-api-operator/pull/368/files/5d2b38bc7a0e838f2032a29c6efb7c966c256912#diff-56c9657f199776b5f04119428ca26c17R8